sys/net/network_layer/ng_ipv6/hdr/ng_ipv6_hdr.c

```diff
@@ -55,6 +55,8 @@ ng_pktsnip_t *ng_ipv6_hdr_build(ng_pktsnip_t *payload,
 
     hdr = (ng_ipv6_hdr_t *)ipv6->data;
 
+    memset(&hdr->src + src_len, 0, sizeof(ng_ipv6_addr_t) - src_len);
```

Comment:
I'd prefer

1. the usage of `ng_ipv6_addr_set_unspecified()` since it is clearer what is meant, it utilizes 64-bit integer assignment instead of looping over an 8-bit array (and is (stated without actually testing it) faster because of it), and it is consistent with the use in the rest of `ng_ipv6`
2. have it done in a newly introduced else-branch to the `if`-check below.